cleanlab/outlier.py

```diff
@@ -0,0 +1,535 @@
+    # neighbor of each point is the point itself, at a distance of zero.
+    distances, _ = knn.kneighbors(features)
+
+    # Calculate average distance to k-nearest neighbors
+    avg_knn_distances = distances[:, :k].mean(axis=1)
+
+    # Map ood_features_scores to range 0-1 with 0 = most concerning
+    ood_features_scores: np.ndarray = np.exp(-1 * avg_knn_distances * t)
+    return (ood_features_scores, knn)
+
+
+def _get_ood_predictions_scores(
+    pred_probs: np.ndarray,
+    *,
+    labels: Optional[np.ndarray] = None,
+    confident_thresholds: Optional[np.ndarray] = None,
+    adjust_pred_probs: bool = True,
+    method: str = "entropy",
+) -> Tuple[np.ndarray, Optional[np.ndarray]]:
+    """Returns an OOD (out of distribution) score for each example based on it pred_prob values.
+
+    Parameters
+    ----------
+    pred_probs : np.ndarray
+      An array of shape ``(N, K)`` of model-predicted probabilities,
+      `pred_probs` in the same format expected by the :py:func:`fit <cleanlab.outlier.OutOfDistribution.fit>` function.
+
+    confident_thresholds : np.ndarray, default = None
+      For details, see key `confident_thresholds` in the params dict arg of :py:class:`OutOfDistribution <cleanlab.outlier.OutOfDistribution>`.
+
+    labels : np.ndarray, optional
+      `labels` in the same format expected by the :py:func:`fit <cleanlab.outlier.OutOfDistribution.fit>` function.
+
+    adjust_pred_probs : bool, True
+      Account for class imbalance in the label-quality scoring.
+      For details, see key `adjust_pred_probs` in the params dict arg of :py:class:`OutOfDistribution <cleanlab.outlier.OutOfDistribution>`.
+
+    method : {"entropy", "least_confidence"}, default="entropy"
+      OOD scoring method.
+      For details see `method` in :py:func:`fit <cleanlab.outlier.OutOfDistribution.fit>` function.
+
+
+    Returns
+    -------
+    ood_predictions_scores : Tuple[np.ndarray, Optional[np.ndarray]]
+      Returns a tuple. First element is array of `ood_predictions_scores` and second is an np.ndarray of `confident_thresholds` or None is 'confident_thresholds' is not calculated.
+    """
+
+    valid_methods = [
+        "entropy",
+        "least_confidence",
+    ]
+
+    if (confident_thresholds is not None or labels is not None) and not adjust_pred_probs:
+        warnings.warn(
+            f"OOD scores are not adjusted with confident thresholds. If scores need to be adjusted set "
+            f"adjusted_pred_probs = True. Otherwise passing in confident_thresholds and/or labels does not change "
+            f"score calculation.",
+            UserWarning,
+        )
+
+    if adjust_pred_probs:
+        if confident_thresholds is None:
+            if labels is None:
+                raise ValueError(
+                    f"Cannot calculate adjust_pred_probs without labels. Either pass in labels parameter or set "
+                    f"adjusted_pred_probs = False. "
```

Comment:
should be `params["adjust_pred_probs"]` instead of `adjusted_pred_probs`.  This should be fixed throughout, you may be using that wrong name in other places as well

Reply:
"labels parameter" 
->
"parameter \`labels\`"